packages/investor-idle/src/IdleInvestor.ts

```diff
@@ -0,0 +1,90 @@
+import Web3 from 'web3'
+// import https from 'https'
+import { find } from 'lodash'
+import filter from 'lodash/filter'
+import { IdleSdk } from './IdleSdk'
+import { Contract } from 'web3-eth-contract'
+import { Logger } from '@shapeshiftoss/logger'
+import { Investor } from '@shapeshiftoss/investor'
+import { KnownChainIds } from '@shapeshiftoss/types'
+import type { ChainAdapter } from '@shapeshiftoss/chain-adapters'
+import { PreparedTransaction, IdleOpportunity } from './IdleOpportunity'
+import { IdleVault, ssRouterAbi, ssRouterContractAddress } from './constants'
+
+type ConstructorArgs = {
+  chainAdapter: ChainAdapter<KnownChainIds.EthereumMainnet>
+  dryRun?: true
+  network?: number
+  providerUrl: string
+}
+
+// const makeRequest = async (url: string): Promise<string> =>  {
```

Comment:
for production can we please use axios for consistency with the rest of this repo?